src/ducks/myApps/index.js

```diff
@@ -90,6 +90,15 @@ export function fetchApps () {
 export function uninstallApp (slug) {
   return (dispatch, getState) => {
     return cozy.client.fetchJSON('DELETE', `/apps/${slug}`)
+    .then(() => {
+      return dispatch({
+        type: 'SEND_LOG_SUCCESS',
+        alert: {
+          message: 'app_modal.uninstall.alerts.success',
```

Comment:
To me it's weird to send an `alert`, an UI related information, from this.

IMHO we should instead have a part of the state reacting to a list of actions, instead of passing those kind of parameters from here.

Reply:
Or maybe `alert` is too much here, we only need `message`.